package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "react-native-helpscout-beacon",
   "title": "React Native Helpscout Beacon",
-  "version": "0.5.0",
+  "version": "0.6.0",
```

Comment:
Sorry, missed this in my last review, the version bump should be part of the release version commit